### PR TITLE
feat(saved-queries): Starting implementation for Saved Queries

### LINKED ISSUE #783: Implement frontend for initial saved queries functionality
(Copied from Notion, needs links to Figma)

****************Frontend****************

- A new section in the `Queries` part of the navigation to list all saved queries and management of them (delete button where relevant, link to “open” a saved query, etc.)
- TODO

### PR DESCRIPTION
**Pull request by german-mergestat**
Starting implementation for `Saved Queries`. (Resolves #783)

- Implementing first approach for Saved queries.
- Fixing bug for empty repository (no showing image).

saved-queries

### CODE REVIEW

**Comment by emilyvandewalle** [comment]
Looks good! I would keep the icons on the top right the same size (20 by 20 pixels), maybe we could use muted borderless `isIconOnly` buttons?

**Comment by german-mergestat** [comment]
> 

@emilyvandewalle you're right, I will change them then... thanks :) 👍

File: ui/src/views/saved-queries/components/saved-queries-table.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  }
+
+  return (
+    <div className='flex flex-col flex-1'>
+      {savedQueries.length < 1
+        ? <div className='flex justify-center py-5'>
+          <Panel className='rounded-md w-full shadow-sm mx-8'>
+            <Panel.Body className='p-0'>
+              <div className='flex justify-center items-center bg-white py-5'>
+                Couldn&#39;t find any saved query!
```
Comment (patrickdevivo) [inline ui/src/views/saved-queries/components/saved-queries-table.tsx:33]:
```suggestion
                Couldn&#39;t find any saved queries.
```

File: ui/src/views/saved-queries/index.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            ? <Button
+              className='whitespace-nowrap justify-center'
+              label='Query'
+              onClick={gotToQueryEditor}
+            />
+            : <Button
+              className='whitespace-nowrap justify-center'
+              label='Add Query'
+              startIcon={<PlusIcon className='t-icon' />}
+              onClick={gotToQueryEditor}
```
Comment (patrickdevivo) [inline ui/src/views/saved-queries/index.tsx:62]:
```suggestion
              label='Query'
              onClick={gotToQueryEditor}
```

```suggestion
              label='Query'
```

To be consistent with the other button when there are existing queries

**Review by patrickdevivo — CHANGES_REQUESTED** [review]

File: ui/src/views/saved-queries/components/saved-queries-table.tsx (hunk 1 above)
Comment (german-mergestat) [reply ui/src/views/saved-queries/components/saved-queries-table.tsx]: Done [4af6128](https://github.com/mergestat/mergestat/pull/807/commits/4af6128fd478dd48a4c05b9d96db399ff2a590f3)

File: ui/src/views/saved-queries/index.tsx (hunk 2 above)
Comment (german-mergestat) [reply ui/src/views/saved-queries/index.tsx]: Done [4af6128](https://github.com/mergestat/mergestat/pull/807/commits/4af6128fd478dd48a4c05b9d96db399ff2a590f3)

**Review by patrickdevivo — APPROVED** [review]

**Pull request merged**